Tasks/CopyFilesV2/copyfiles.ts

```diff
@@ -11,6 +11,7 @@ let targetFolder: string = tl.getPathInput('TargetFolder', true);
 let cleanTargetFolder: boolean = tl.getBoolInput('CleanTargetFolder', false);
 let overWrite: boolean = tl.getBoolInput('OverWrite', false);
 let flattenFolders: boolean = tl.getBoolInput('flattenFolders', false);
+let keepTimestamp: boolean = tl.getBoolInput('keepTimestamp', false);
```

Comment:
Nit: preserveTimestamp? More descriptive of what it's doing, IMO.

Reply:
I've changed this to preserve that seems like more clear language than keep or update.